cl/antiquary/tests/tests.go

```diff
@@ -128,6 +141,34 @@ func LoadChain(blocks []*cltypes.SignedBeaconBlock, s *state.CachingBeaconState,
 	return m
 }
 
+func GetElectraRandom() ([]*cltypes.SignedBeaconBlock, *state.CachingBeaconState, *state.CachingBeaconState) {
+	block1 := cltypes.NewSignedBeaconBlock(&clparams.MainnetBeaconConfig, clparams.ElectraVersion)
+	block2 := cltypes.NewSignedBeaconBlock(&clparams.MainnetBeaconConfig, clparams.ElectraVersion)
+
+	// Lets do te
+	if err := utils.DecodeSSZSnappy(block1, electra_blocks_0_ssz_snappy, int(clparams.ElectraVersion)); err != nil {
+		panic(err)
+	}
+	if err := utils.DecodeSSZSnappy(block2, electra_blocks_1_ssz_snappy, int(clparams.ElectraVersion)); err != nil {
+		panic(err)
+	}
+	fmt.Println("shota 1")
+	preState := state.New(&clparams.MainnetBeaconConfig)
+	if err := preState.UpgradeToElectra(); err != nil {
+		panic(err)
+	}
+	if err := utils.DecodeSSZSnappy(preState, electra_pre_state_ssz_snappy, int(clparams.ElectraVersion)); err != nil {
```

Comment:
Version should be Deneb here

Reply:
oh and here is a tricky stuff.
because we don't know the fork epoch of electra, so need to manually put it back to beacon config
eg: `preState.BeaconConfig().ElectraForkEpoch = meta.ForkEpoch`, where meta is from meta.yaml

Reply:
yes, because the upgrade function need this value to do fork